pages/docs/tutorials/native/using-intellij-idea.md

```diff
@@ -1,50 +1,131 @@
 ---
 type: tutorial
 layout: tutorial
-title:  "Hello Kotlin/Native using IntelliJ IDEA"
-description: "A look at how to create a Kotlin/Native application using IntelliJ IDEA"
-authors: Hadi Hariri
-date: 2020-01-15
+title:  "Get started with Kotlin/Native using IntelliJ IDEA"
+description: "This tutorial demonstrates how to use IntelliJ IDEA for creating a Kotlin/Native application."
+authors: Hadi Hariri, Andrey Polyakov
+date: 2020-08-20
+showAuthorInfo: false
 ---
 
 <!--- To become a How-To. Need to change type to new "HowTo" --->
 
+To get started, install the latest version of [IntelliJ IDEA](http://www.jetbrains.com/idea/download/index.html). The tutorial is applicable to both IntelliJ IDEA Community Edition and the Ultimate Edition.
 
-## Creating a new Kotlin/Native project in IntelliJ IDEA
+## Create a new Kotlin/Native project in IntelliJ IDEA
 
-This following is applicable to both [IntelliJ IDEA Community Edition as well as the Ultimate Edition](https://www.jetbrains.com/idea).
+1. In IntelliJ IDEA, select **File** \| **New** \| **Project**.
+2. In the panel on the left, select **Kotlin**.
+3. Enter a project name, select **Native Application** as the project template, and click **Next**.
 
+   ![Create a native application]({{ url_for('tutorial_img', filename='native/using-intellij-idea/native-new-project-intellij-1.png') }})
 
-From the **File** menu in IntelliJ IDEA or **Welcome screen**, select to create a new Project and in the first step of the Wizard
-select **Kotlin** on the left hand column and **Native | Gradle** on the right hand.
+   By default, your project will use Gradle with Kotlin DSL as the build system.
+   > Kotlin/Native doesn't support Maven and IntelliJ build systems.
+   {:.note}
 
-![Wizard Step One]({{ url_for('tutorial_img', filename='native/using-intellij-idea/wizard.png')}})
+4. Accept the default configuration on the next screen and click **Finish**.
 
-Click **Next** in the dialog and in the next step make sure **Automatically import this project on changes in build script** is checked. This is useful when starting
-out to make sure any immediate changes to the build script are imported automatically. 
+   ![Configure a native application]({{ url_for('tutorial_img', filename='native/using-intellij-idea/native-new-project-intellij-2.png') }})
 
-![Wizard Step Two]({{ url_for('tutorial_img', filename='native/using-intellij-idea/wizard-2.png')}})
+Your project opens. By default, the wizard creates the necessary `main.kt` file with the code that prints "Hello, Kotlin/Native!" to the standard output.
 
-On clicking **Next**, enter the path and name for the project.
+The `build.gradle.kts` file contains the project settings. You can change the settings using [Kotlin Multiplatform Gradle DSL reference](../../reference/mpp-dsl-reference.html).
```

Comment:
What settings do you recommend changing in this tutorial? DSL Reference is too long and pretty complicated to get started.

Reply:
I changed the sentence to "Read more about the settings...".